src/test/src/sc/fiji/bdvpg/bdv/sourceAndConverter/SourceAndConverterLoaderDemo.java

```diff
@@ -1,37 +1,34 @@
 package sc.fiji.bdvpg.bdv.sourceAndConverter;
 
 import bdv.util.BdvHandle;
-import bdv.util.BdvStackSource;
 import bdv.viewer.SourceAndConverter;
 import sc.fiji.bdvpg.bdv.BdvCreator;
+import sc.fiji.bdvpg.bdv.navigate.ViewerTransformAdjuster;
 import sc.fiji.bdvpg.bdv.source.append.SourceAndConverterBdvAdder;
 import sc.fiji.bdvpg.sourceandconverter.SourceAndConverterLoader;
+import sc.fiji.bdvpg.sourceandconverter.display.BrightnessAdjuster;
 
 public class SourceAndConverterLoaderDemo
 {
 	public static void main( String[] args )
 	{
-		final BdvCreator bdvCreator = new BdvCreator( false );
-		bdvCreator.run();
-		final BdvHandle bdvHandle = bdvCreator.get();
+		final BdvHandle bdvHandle = createBdv();
+		final SourceAndConverter< ? > sourceAndConverter = loadSourceAndConverter();
+		new SourceAndConverterBdvAdder( bdvHandle, sourceAndConverter ).run();
+		new ViewerTransformAdjuster( bdvHandle, sourceAndConverter.getSpimSource() ).run();
+		new BrightnessAdjuster( bdvHandle, sourceAndConverter, 0, 255.0 ).run();
+	}
 
-		// load
-		//
+	public static SourceAndConverter< ? > loadSourceAndConverter()
+	{
 		final SourceAndConverterLoader loader = new SourceAndConverterLoader( "src/test/resources/mri-stack.xml" );
-		final SourceAndConverter< ? > sourceAndConverter = loader.getSourceAndConverter( 0 );
-
-		// add
-		//
-		final SourceAndConverterBdvAdder adder = new SourceAndConverterBdvAdder( bdvHandle, sourceAndConverter );
-		adder.run();
-
-		// change brightness
-		//
-		final BdvStackSource< ? > bdvStackSource = adder.getBdvStackSource();
-		bdvStackSource.setDisplayRange( 0, 255 );
+		return loader.getSourceAndConverter( 0 );
```

Comment:
The output are supposed to not have any parameter according to our design. Returning a List of all source and converter could be better, also it avoids calling BigDataViewer.init as many times as there are sources you want to get.